servers/src/main/java/tachyon/master/next/block/BlockMaster.java

```diff
@@ -146,7 +163,9 @@ public void writeToJournal(JournalOutputStream outputStream) throws IOException
   public void start(boolean asMaster) throws IOException {
     startMaster(asMaster);
     if (isMasterMode()) {
-      // TODO: start periodic heartbeat threads.
+      mHeartbeat = getExecutorService()
+          .submit(new HeartbeatThread("Master Heartbeat", new MasterInfoHeartbeatExecutor(),
```

Comment:
I think the name should be more descriptive like "Lost Worker Detector" or something.